www/plas02/styless_ukol4.css

```diff
@@ -0,0 +1,127 @@
+*{
+    font-family: sans-serif;
+}
+
+.red_text {
+    color: red;
+}
+
+.green_text {
+    color: rgb(47, 235, 41);
+}
+
+.blue_text {
+    color: #3399ff;
+    font-weight: bold;
+}
+
+.small_text {
+    font-size: 18px;
+}
+
+.big_text {
+    font-size: 24px;
+    font-style: italic;
+}
+
+.background_text {
+    background: brown;
+}
+
+.small_p{
+    font-size: 32px;
+    text-align: center;
+}
+
+.capital_p {
+
+    text-align: justify;
+    text-transform: uppercase; 
+}
+
+.white_box {
+    height: 300px;
+    width: 400px;
+    border-top: 1px solid purple;
+    border-right: 1px solid purple;
+    border-bottom: 1px solid purple;
+    border-left: 1px solid purple;
+    padding: 30px;
+}
+
+.red_box {
+    height: 200px;
+    width: 500px;
+    background: red;
+    margin: 100px;
+}
+
+.invisible_text{
+    color: white;
+}
+
+.blue_box {
+    height: 150px;
+   
+    background: blue;
+    padding: 50px;
+
+    border-top: 10px solid yellow;
+    border-right: 10px solid yellow;
+    border-bottom: 10px solid yellow;
+    border-left: 10px solid yellow;
+}
+
+.gap {
+    color:white;
+}
+
+.green_box {
+    width: 200px;
+    height: 500px;
+    background: rgb(14, 218, 123);
+    opacity: 0.5;
+}
+
+#light_box {
+    width: 300px;
+    height: 400px;
+    background: rgb(98, 193, 231);
+    position: relative;
+}
+
+#light_box #pink_box {
+    width: 50%;
```

Comment:
width: 50px; height: 50px;

a kdyz uvedes vsechny top left right bottom tak to nebude fungovat, protoze nemuze byt top 50px a zaroven bottom: 0, tak jako nemuze byt right 10px a zaroven left 0, takze left a bottom smaz